packages/ckb-sdk-utils/src/crypto/blake2b.ts

```diff
@@ -374,10 +375,24 @@ export const blake2b = (
   return new Blake2b(outlen, key, salt, personal)
 }
 
-export const ready = (cb: Function) => {
-  b2wasm.ready(() => {
-    cb()
-  })
+export const ready = (cb: Function) => {  
+  const ua = navigator.userAgent.toLowerCase()
+  const IOSVersion = ua.match(/cpu iphone os (.*?) like mac os/)
+  
+  // ios11 is not fully compatible wasm and change model to blakejs
+  if (IOSVersion && IOSVersion[1].split('_')[0] === 11) {
```

Comment:
`IOSVersion[1].split('_')[0]` is a string

Reply:
Error check fixed